docs/en/reference/sharding_azure_tutorial.rst

```diff
@@ -26,19 +26,21 @@ Install Doctrine
 
 For this tutorial we will install Doctrine and the Sharding Extension through
 `Composer <http://getcomposer.org>`_ which is the easiest way to install
-Doctrine. Composer is a new package manager for PHP. Download the
+Doctrine. Composer is a package manager for PHP. Download the
 ``composer.phar`` from their website and put it into a newly created folder for
 this tutorial. Now create a ``composer.json`` file in this project root with
 the following content:
 
+.. code-block:: json
+
     {
         "require": {
             "doctrine/dbal": "2.2.2",
             "doctrine/shards": "0.2"
         }
     }
 
-Open up the commandline and switch to your tutorial root directory, then call
+Open up the command-line and switch to your tutorial root directory, then call
```

Comment:
Should it be "command line"? Command-line is adjective.

Reply:
Fixed.